conf/pacific/cephadm/disconnected_installation.yaml

```diff
@@ -0,0 +1,55 @@
+---
```

Comment:
Going down the path of not having `network-node`... I believe we can avoid this configuration and reuse the existing ones.